scripts/exrrfsfw_domain.py

```diff
@@ -0,0 +1,242 @@
+#!/bin/env python
```

Comment:
@BenjaminBlake-NOAA, can you move this script to `ush` and rename it (without exrrfs)?

Reply:
@chan-hoo  Sure I can do that.  I thought NCO wanted fewer scripts/files in the ush directory though?  But it makes sense to keep this Python script in ush since it is called by an ex-script.